Playmate/Utilities/Constants.m

```diff
@@ -222,46 +259,32 @@ + (NSArray *)sportsList:(BOOL)needAll {
     
     NSMutableArray *sports = [[NSMutableArray alloc] init];
     
-    // TODO: Need to debug this for getting all sports later
-    /*
+    if (needAll) [sports addObject:@"All"];
+    
     APIManager *manager = [APIManager new];
     [manager getSportsListWithCompletion:^(NSDictionary *list, NSError *error) {
         if (error != nil) {
             NSLog(@"%@", error.localizedDescription);
         } else {
             NSArray *data = list[@"data"];
-
+            
             for (NSDictionary *datum in data) {
-                NSString *sport = [NSString stringWithString:datum[@"attributes"][@"name"]];
-
-                NSString *sportType = [NSString stringWithFormat:@"%@", [sport class]];
-
-                if ([sportType isEqualToString:@"NSTaggedPointerString"]) {
-                    [sports addObject:sport];
-                }
+                
+                NSString *sport = [NSString stringWithFormat:@"%@", datum[@"attributes"][@"name"]];
+                [sports addObject:sport];
             }
         }
     }];
-
-    NSLog(@"%@", sports);
-    */
-    
-    if (needAll) {
-        [sports addObject:@"All"];
-    }
     
-    [sports addObject:@"Tennis"];
-    [sports addObject:@"Basketball"];
-    [sports addObject:@"Golf"];
     return (NSArray *)sports;
```

Comment:
I like more to do this: 
`[NSArray arrayWithArray:mutableArray]`

This way you really return a NSArray, instead of a NSMutableArray that is just casted as NSArray. 

It is super minor, but a little bit more correct.